### PR TITLE
Fixes #22269 - Added consistent order to os templates

### PR DESCRIPTION
**Pull request by yifatmakias**
the templates are ordered by the template_kind_id.

### CODE REVIEW

**Comment by tbrisker** [comment]
ok to test

File: app/views/operatingsystems/_template_defaults.html.erb [hunk 1]
```diff
@@ -11,8 +11,7 @@
                 :text => (_("You probably need to configure your %s first.") %
                           link_to("templates", provisioning_templates_path)).html_safe) %>
     <% end %>
-
-    <%= f.fields_for :os_default_templates do |builder| %>
+    <%= f.fields_for :os_default_templates, @operatingsystem.os_default_templates.order('template_kind_id') do |builder| %>
```
Comment (lzap) [inline app/views/operatingsystems/_template_defaults.html.erb]: I'd probably rather join tables and sort by name. But I am OK with this, only if you want to spend more time with this.

**Review by lzap — DISMISSED** [review]
It's green, tell us if you want to do the join or not. Thanks!

**Comment by ares** [comment]
@yifatmakias what is the status here?

File: app/views/operatingsystems/_template_defaults.html.erb (hunk 1 above)
Comment (yifatmakias) [reply app/views/operatingsystems/_template_defaults.html.erb]: I agree I will join and sort by name.

**Review by shiramax — APPROVED** [review]
Thanks @yifatmakias

**Pull request merged**